### ISSUE TITLE
Impaired word retrieval

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6775]
**Preferred term label:**
Impaired word retrieval

**Synonyms**
Word-finding difficulty

**Definition (free text, please give PubMed ID)**
This term is intended to describe the fact that some persons with long COVID describe difficulty finding words while speaking or writing (https://www.medrxiv.org/content/10.1101/2020.12.24.20248802v2). Clinically, this is different from Motor aphasia (https://hpo.jax.org/app/browse/term/HP:0002427), however it is difficult to provide a precise definition. Here is a useful article that discusses the differential diagnosis but does not contain a definition
https://academic.oup.com/brain/article/131/1/8/342737

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@DLewisSmith do you happen to have advice about this? Is "Word-finding difficulty" basically always "(Motor) Apahasia" of a certain degree of severity or is there a case to be made for having a separate term?

**Comment by DLewisSmith** [comment]
I recall the Rohrer paper as the best instructive text on this complex topic. Impaired word retrieval may be acceptable as a type of Aphasia (I am not sure that I would specify Motor aphasia as this is really an anomia) but I worry that Word-finding difficulty is too vague. It is a commonly used description for people struggling to answer questions or generate speech for a variety of reasons that need careful disentanglement.

@meherlad what do you think, can you advise how best to structure the relationship between different concepts in the aphasias? Happy to discuss via phone.

I would have thought something such as:
Aphasia has children: Productive/Motor/Broca, Receptive/Sensory/Wernicke, Conduction (may be a bit outdated), Would you include Semantic paraphrasia?

**Comment by pnrobinson** [comment]
I think for now we modeled this as aphasia--word finding difficulty is probably too vague. We can revist this if needed.
